spec/listformat.html

```diff
@@ -383,14 +389,7 @@ <h1>Properties of Intl.ListFormat Instances</h1>
       <li>[[Locale]] is a String value with the language tag of the locale whose localization is used by the list format styles.</li>
       <li>[[Type]] is one of the String values `"conjunction"`, `"disjunction"`, or `"unit"`, identifying the list of types used.</li>
       <li>[[Style]] is one of the String values `"long"`, `"short"`, or `"narrow"`, identifying the list formatting style used.</li>
-      <li>[[TemplatePair]] is a template for lists of size two. It is expected to be a valid template pattern, as described below.</li>
-      <li>[[TemplateStart]] is a template for lists of size greater than two, to be used at the start of the output. It is expected to be a valid template pattern, as described below.</li>
-      <li>[[TemplateMiddle]] is a template for lists of size greater than two, to be used in the middle of the output. It is expected to be a valid template pattern, as described below.</li>
-      <li>[[TemplateEnd]] is a template for lists of size greater than two, to be used at the end of the output. It is expected to be a valid template pattern, as described below.</li>
+      <li>[[Templates]] is a ListFormat template set.</li>
```

Comment:
Would this automatically link to `<dfn>ListFormat template set</dfn>`?

Reply:
I think so?

Reply:
Okay, great. Just wanted to make sure. :smile: